specifyweb/frontend/js_src/lib/localization/tree.ts

```diff
@@ -357,4 +357,7 @@ export const treeText = createDictionary({
     'fr-fr': 'Modifier les rangs',
     'uk-ua': 'Редагувати ранги',
   },
+  collapseAll: {
```

Comment:
The localization tests are failing right now. Please take a look at the error message they give you

<details>
<summary>(Spoiler) Solution</summary>
common.ts dictionary already declares a `collapseAll` key, so you don't have to define it again in tree.ts
</details>